recipes-tag/16-id-lix-analysis/meta.yaml

```diff
@@ -12,3 +12,6 @@ build:
 requirements:
   run:
     - fabio
+    - py4xs
+    - lmfit >=0.9.9
+    - peakutils
```

Comment:
Peakutils is a general dependency of the `analysis` package and does not need to be included.